layers/+web/eww/funcs.el

```diff
@@ -90,16 +114,16 @@ full hint text will not show up!"
     ("u" eww-up-url)
     ("t" eww-top-url)
     ;; Layout/Appearance
-    ;; ("w" writeroom-mode)
+    ("w" identity)
```

Comment:
`identity` is not a valid key binding definition.  If the user presses this key and they are not using the `writeroom-mode` package this errors:

```
wrong type argument: commandp, identity
```